src/Behat/Gherkin/Filter/TagFilter.php

```diff
@@ -137,6 +138,10 @@ protected function isTagsMatchCondition($tags)
     {
         $satisfies = true;
 
+        if (mb_strlen($this->filterString) === 0) {
```

Comment:
no need to use `mb_strlen`. Using `\strlen(...)` will be faster (PHP stores the length of strings so this would be a `O(1)` operation while `mb_strlen` is `O(n)`).

Reply:
Done